docs/platform-services/automation-service/app-central/integrations/slack.md

```diff
@@ -99,9 +99,11 @@ Create a public or private channel, Send messages or Files to channels or Users
 1. The integration details will appear. Click on the **"+"** button to add new Resource.<br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/slack/slack-13.png')} style={{border:'1px solid gray'}} alt="slack" width="200"/>
 1. Create a new resource for the User.
 1. Copy the User OAuth Token from Slack webpage and paste it In the Bot/User OAuth Token.
+1. If using an Org-level Token, select the Token Type as Org-level Token and provide the Team ID. By default, it is set to Workspace-Level Token, which does not require a Team ID.
 1. Once you have filled in all the required fields, click **SAVE**.<br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/slack/slack-14.png')} style={{border:'1px solid gray'}} alt="slack" width="400"/>
 1. Create another resource for the Bot User.
 1. Copy the Bot User OAuth Token from Slack webpage and paste it In the Bot/User OAuth Token.
+1. If using an Org-level Token, select the Token Type as Org-level Token and provide the Team ID. By default, it is set to Workspace-Level Token, which does not require a Team ID.
```

Comment:
Is this step supposed to be here twice?

Reply:
Yes, as the user will be configuring both user and bot OAuth token so for both we need to add above statement.